### ISSUE TITLE
unexpected record `BRF`

### ISSUE THREAD

**Issue by PSeitz** [gifnksm/lcov#31]
with llvm-cov there are BRF entries which the tool does not understand

Reproducable on lz4_flex https://github.com/PSeitz/lz4_flex
```
cargo llvm-cov --workspace --lcov --output-path lcov1.info
cargo llvm-cov --no-default-features --features frame --features checked-decode --workspace --lcov --output-path lcov3.info

lcov-merge lcov1.info lcov3.info
```

**Comment by gifnksm** [comment]
Parsing failed because the order of the records in the info file generated by `llvm-cov` was unexpected.

`lcov-merge` assumes that the records are sorted in the same order as described in [the `FILES` section of `geninfo(1)`](https://man.archlinux.org/man/geninfo.1#FILES).
Therefore, `lcov-merge` expects the` BRF` record to appear *before* the `DA` record.
The information file generated by `llvm-cov` seems to have a` BRF` record *after* the `DA` record.

I will try to modify the program so that it can be analyzed even if the order of the records is out of order.

**Comment by gifnksm** [comment]
released as [lcov-util v0.1.6](https://crates.io/crates/lcov-util)
